java/src/jmri/jmrix/cmri/serial/sim/configurexml/ConnectionConfigXml.java

```diff
@@ -33,6 +33,11 @@ protected void getInstance() {
     }
 
     @Override
+    protected void getInstance(Object object) {
```

Comment:
I found that after I asked. Makes me wonder if the super class should take a ConnectionConfig instead of an Object since I can't find an implementation of that method that does not cast object to a ConnectionConfig or subclass of ConnectionConfig.